posix_compliance/client.py

```diff
@@ -0,0 +1,72 @@
+# Prepare a NFS-client for posix compliance testing
+# - install needed tools and libs
+# - checkout posix compliance from git
+# - build posix compliance test suite
+# - run the NFSv4.0 tests
+#
+
+import os
+import subprocess
+import time
+import sys
+
+#get the environment variables
+server=os.getenv("SERVER")
+export=os.getenv("EXPORT")
+
+#Install required packages for posix compliance test suite
+cmd = "yum -y install git gcc nfs-utils redhat-rpm-config python-devel krb5-devel perl-Test-Harness *libacl*"
+subprocess.call(cmd, shell=True)
+
+#install posix compliance test suite
+cmd = "rm -rf /root/ntfs-3g-pjd-fstest && git clone https://github.com/ffilz/ntfs-3g-pjd-fstest.git"
```

Comment:
Maybe place the path for this git repo as a variable in the top of the script?
I know Zorro Lang (zlang@redhat.com) does some tests and corrections to the upstream project too. Is that different from the repository from Frank Filz?